quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -305,12 +306,12 @@ private static void addFeatureOptions(CommandSpec commandSpec) {
                 .order(ConfigCategory.FEATURE.getOrder())
                 .validate(false);
 
-        Set<String> featuresExpectedValues = Arrays.stream(Profile.Type.values()).map(type -> type.name().toLowerCase()).collect(Collectors.toSet());
+        String previewName = Profile.Type.PREVIEW.name().toLowerCase();
 
         featureGroupBuilder.addArg(OptionSpec.builder(new String[] {"-ft", "--features"})
-                .description("Enables a group of features. Possible values are: " + String.join(",", featuresExpectedValues))
-                .paramLabel("feature")
-                .completionCandidates(featuresExpectedValues)
+                .description("Enables all tech preview features.")
+                .paramLabel(Profile.Type.PREVIEW.name().toLowerCase())
```

Comment:
A nitpick, but since you have the `previewName` local var perhaps you should just use it here too?

Reply:
just to make sure: you mean also use it in the description? Can do for sure